LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
I'd probably assume just go with the MIT License. https://choosealicense.com/licenses/mit/. Also, the file convention on GitHub is `LICENSE.md`. 🙇

Reply:
This is the license I found in the file, per the original issue. But I can change it if you'd like.

Reply:
Ah..fair enough..it's been a while since having looked at that header :smile:. If you can just rename this to `LICENSE.md` we should be good to go.